pkg/generators/forwarding/fluentd/loki_conf.go

```diff
@@ -0,0 +1,50 @@
+package fluentd
+
+import (
+	"fmt"
+	"strings"
+)
+
+var defaultLokiLabelKeys = []string{
+	"log_type",
+	"kubernetes.namespace_name",
+	"kubernetes.pod_name",
```

Comment:
I don't think we should keep `kubernetes.pod_name` as a default label. Since openshift/k8s clusters can have constantly changing pod names this might lead to a high cardinality and reduce querying performance ([Docs](https://grafana.com/blog/2020/04/21/how-labels-in-loki-can-make-log-queries-faster-and-easier/#cardinality)).